arch/arm/core/aarch32/thread.c

```diff
@@ -353,6 +353,11 @@ int arch_float_disable(struct k_thread *thread)
 
 	return 0;
 }
+
+int arch_float_enable(struct k_thread *thread)
+{
+	return -ENOSYS;
```

Comment:
That's fine for me. Please, add an inline comment that "this is not supported in Cortex-M" or something similar

Reply:
FYI: Cortex-R does not support FPU in general, at the moment.

Reply:
OK, added such information.